server/channels/app/syncables.go

```diff
@@ -253,17 +266,25 @@ func (a *App) SyncRolesAndMembership(c request.CTX, syncableID string, syncableT
 	switch syncableType {
 	case model.GroupSyncableTypeTeam:
 		params.ScopedTeamID = &syncableID
-		a.createDefaultTeamMemberships(c, params)
-		a.deleteGroupConstrainedTeamMemberships(c, &syncableID)
+		if err := a.createDefaultTeamMemberships(rctx, params); err != nil {
+			rctx.Logger().Warn("Error creating default team memberships", mlog.Err(err))
```

Comment:
I think we should log these as errors instead of warnings. Considering we are also literally writing "Error" in the "warning" log.

Reply:
I don't think they should be the Errors in the logs. There is nothing imminently actionable about them. It is about informing the admin that something went wrong.